diff --git a/components/net/lib.rs b/components/net/lib.rs
index cc013f0a3c3..3c6e0eaf857 100644
--- a/components/net/lib.rs
+++ b/components/net/lib.rs
@@ -23,6 +23,9 @@ pub mod resource_thread;
 pub mod subresource_integrity;
 #[cfg(feature = "test-util")]
 pub mod test_util;
+pub mod tor_connector;
+pub mod transport_url;
+pub mod unix_connector;
 mod websocket_loader;
 
 /// An implementation of the [Fetch specification](https://fetch.spec.whatwg.org/)
