diff --git a/components/net/lib.rs b/components/net/lib.rs
index e531021faf5..5522f7647b6 100644
--- a/components/net/lib.rs
+++ b/components/net/lib.rs
@@ -20,6 +20,9 @@ pub mod protocols;
 pub mod request_interceptor;
 pub mod resource_thread;
 pub mod subresource_integrity;
+pub mod tor_connector;
+pub mod transport_url;
+pub mod unix_connector;
 mod websocket_loader;
 
 /// An implementation of the [Fetch specification](https://fetch.spec.whatwg.org/)
