diff --git a/components/net/Cargo.toml b/components/net/Cargo.toml
index 94061f2ae18..527b59d6f50 100644
--- a/components/net/Cargo.toml
+++ b/components/net/Cargo.toml
@@ -21,6 +21,8 @@ test-util = ["hyper-util/server-graceful"]
 async-compression = { version = "0.4.12", default-features = false, features = ["brotli", "gzip", "tokio", "zlib", "zstd"] }
 async-recursion = "1.1"
 async-tungstenite = { workspace = true }
+aws-lc-rs = { workspace = true }
+tokio-socks = "0.5"
 base = { workspace = true }
 base64 = { workspace = true }
 bytes = "1"
@@ -42,6 +44,7 @@ http-body-util = { workspace = true }
 hyper = { workspace = true, features = ["client", "http1", "http2"] }
 hyper-rustls = { workspace = true }
 hyper-util = { workspace = true }
+hyperlocal = "0.9"
 hyper_serde = { workspace = true }
 imsz = { workspace = true }
 ipc-channel = { workspace = true }
